CONTRIBUTING.md

```diff
@@ -1,15 +1,45 @@
-# Contributing to Anchor
```

Comment:
What's the thought process for changing the headline? I prefer "Contributing to Anchor" over "Anchor", since this is indeed the contributing document.

Reply:
Similarly for the logo. Seems like we should remove it.

Reply:
removed the headline by mistake